### PR TITLE
https://issues.redhat.com/browse/ACM-13460

### PR DESCRIPTION
**Pull request by jc-berger**
See Jira: https://issues.redhat.com/browse/ACM-13460

### CODE REVIEW

File: business_continuity/backup_restore/backup_arch.adoc [hunk 1]
```diff
@@ -18,7 +18,13 @@ View the following list of the cluster backup and restore process, and how they
 
 - Back up all resources with an API version suffixed by `.open-cluster-management.io` and `.hive.openshift.io`. These suffixes indicate that all {acm-short} resources are backed up.
 
-- Back up all resources from: `argoproj.io`, `app.k8s.io`, `core.observatorium.io`, `hive.openshift.io`. These resources are backed up within the `acm-resources-schedule` backup, with the exception of the resources from the `agent-install.openshift.io` API group. These resources are backed up within the `acm-managed-clusters-schedule` backup.
+- Back up all resources from: `argoproj.io`, `app.k8s.io`, `core.observatorium.io`, `hive.openshift.io`. These resources are backed up within the `acm-resources-schedule` backup, with the exception of the resources from the `agent-install.openshift.io` API group. These resources are backed up within the `acm-managed-clusters-schedule` backup. The following `hive.openshift.io` and `hiveinternal.openshift.io` resources are also backed up by the `}}{{acm-managed-clusters-schedule` backup:
```
Comment (birsanv) [inline business_continuity/backup_restore/backup_arch.adoc]: looks good, one typo : `**}}{{**acm-managed-clusters-schedule` should be `acm-managed-clusters-schedule`

**Review by birsanv — CHANGES_REQUESTED** [review]

File: business_continuity/backup_restore/backup_arch.adoc [hunk 2]
```diff
@@ -18,7 +18,13 @@ View the following list of the cluster backup and restore process, and how they
 
 - Back up all resources with an API version suffixed by `.open-cluster-management.io` and `.hive.openshift.io`. These suffixes indicate that all {acm-short} resources are backed up.
 
-- Back up all resources from: `argoproj.io`, `app.k8s.io`, `core.observatorium.io`, `hive.openshift.io`. These resources are backed up within the `acm-resources-schedule` backup, with the exception of the resources from the `agent-install.openshift.io` API group. These resources are backed up within the `acm-managed-clusters-schedule` backup.
+- Back up all resources from: `argoproj.io`, `app.k8s.io`, `core.observatorium.io`, `hive.openshift.io`. These resources are backed up within the `acm-resources-schedule` backup, with the exception of the resources from the `agent-install.openshift.io` API group. These resources are backed up within the `acm-managed-clusters-schedule` backup. The following `hive.openshift.io` and `hiveinternal.openshift.io` resources are also backed up by the `**}}{{**acm-managed-clusters-schedule should be acm-managed-clusters-schedule` backup:
```
Comment (dockerymick) [inline business_continuity/backup_restore/backup_arch.adoc]: Are these domains, API groups, or resources? @birsanv @jc-berger 
```suggestion
- Back up all resources from the following API groups: `argoproj.io`, `app.k8s.io`, `core.observatorium.io`, `hive.openshift.io`. The resources are backed up within the `acm-resources-schedule` backup, with the exception of the resources from the `agent-install.openshift.io` API group. Resources from the `agent-install.openshift.io` API group are backed up within the `acm-managed-clusters-schedule` backup. The following `hive.openshift.io` and `hiveinternal.openshift.io` resources are also backed up by the `**}}{{**acm-managed-clusters-schedule should be acm-managed-clusters-schedule` backup: 
```

Based on the answer of my previous question:

```suggestion
- Back up all resources from the following API groups: `argoproj.io`, `app.k8s.io`, `core.observatorium.io`, `hive.openshift.io`. The resources are backed up within the `acm-resources-schedule` backup, with the exception of the resources from the `agent-install.openshift.io` API group. Resources from the `agent-install.openshift.io` API group are backed up within the `acm-managed-clusters-schedule` backup. The following resources from `hive.openshift.io` and `hiveinternal.openshift.io` API groups are also backed up by the `acm-managed-clusters-schedule` backup: 
```

**Review by dockerymick — CHANGES_REQUESTED** [review]

File: business_continuity/backup_restore/backup_arch.adoc (hunk 2 above)
Comment (birsanv) [reply business_continuity/backup_restore/backup_arch.adoc]: @dockerymick `hiveinternal.openshift.io` is api group, clusterdeployment.hive.openshift.io is resource

btw, this is incorrect ( the addition of the and the `acm-managed-clusters-schedule` backup )

Incorrect
.. API groups are also backed up by the `**}}{{**acm-managed-clusters-schedule` **and the `acm-managed-clusters-schedule`** backup:

Should be
..API groups are also backed up by the `acm-managed-clusters-schedule`  backup:

Comment (dockerymick) [reply business_continuity/backup_restore/backup_arch.adoc]: Thanks for clarifying, I wasn't sure what the `**}}{{**` was for

Comment (birsanv) [reply business_continuity/backup_restore/backup_arch.adoc]: it was a typo .. :)

**Review by dockerymick — APPROVED** [review]

**Pull request merged**